Formula/t/tombi.rb

```diff
@@ -0,0 +1,36 @@
+class Tombi < Formula
+  desc "TOML Formatter / Linter / Language Server"
+  homepage "https://github.com/tombi-toml/tombi"
+  url "https://github.com/tombi-toml/tombi/archive/refs/tags/v0.6.5.tar.gz"
+  sha256 "4d97f5c740e35b99c28f382e29a3f792241bb756371c41a66b2c64f0f7ee4a0c"
+  license "MIT"
+  head "https://github.com/tombi-toml/tombi.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args(path: "rust/tombi-cli")
+  end
+
+  test do
+    require "open3"
+
+    json = <<~JSON
+      {
+        "jsonrpc": "2.0",
+        "id": 1,
+        "method": "initialize",
+        "params": {
+          "rootUri": null,
+          "capabilities": {}
+        }
+      }
+    JSON
+
+    Open3.popen3("#{bin}/#{name}", "lsp") do |stdin, stdout|
+      stdin.write "Content-Length: #{json.size}\r\n\r\n#{json}"
+      sleep 3
```

Comment:
if 1 is fine, let's use 1
```suggestion
      sleep 1
```

Reply:
tested and it works with 1, i modified it